{{cookiecutter.project_name}}/README.md

```diff
@@ -0,0 +1,47 @@
+<p align="center">
+  <img src="https://raw.githubusercontent.com/dbt-labs/dbt/ec7dee39f793aa4f7dd3dae37282cc87664813e4/etc/dbt-logo-full.svg" alt="dbt logo" width="500"/>
+</p>
+
+**[dbt](https://www.getdbt.com/)** enables data analysts and engineers to transform their data using the same practices that software engineers use to build applications.
+
+dbt is the T in ELT. Organize, cleanse, denormalize, filter, rename, and pre-aggregate the raw data in your warehouse so that it's ready for analysis.
+
+## {{ cookiecutter.project_name }}
+This repo contains the base code to help you start to build out your {{ cookiecutter.project_name }} adapter plugin, for more information on how to build out the adapter please follow the [docs](https://docs.getdbt.com/docs/contributing/building-a-new-adapter)
+
+ ** Note ** this `README` is meant to be replaced with what information would be required to use your adpater once your at a point todo so.
+
+ ## Getting Started
+
+ #### Setting up Locally
+- run `pip install -r dev_requirements.txt`.
+- cd directory into the `dbt-core` you'd like to be testing against and run `make dev`.
+
+ #### Connect to Github
+- run `git init`.
+- Connect your lcoal code to a Github repo.
+
+ ## Testing
+Testing for adapter connection capablilities
+- Once you have your plugin connecting to a database, you can test basic dbt functionality against the by running
+```
+ python -m pytest tests/functional/adapter/test_basic.py
+```
+
+Once your adapter is built out a few options.
+- you can use `dbt debug` to troubleshoot issues with macros
+- [Jaffle_shop](https://docs.getdbt.com/tutorial/setting-up) is a frinedly beginner project to test adapter compatiablity with dbt functionality.
+
```

Comment:
Is this the testing & development order we'd recommend? To my mind, we should be encouraging new adapter development in the following order:
1. Create a toy project (e.g. `jaffle_shop`, https://github.com/dbt-labs/jaffle_shop) to use with their adapter. Install 
2. First, work on the connection methods. Use `dbt debug` to ensure that a basic connection works. (We're missing lower-level unit/functional tests that could help here)
3. Next, work on the higher-level macros/methods. Use the sample project and simple commands (`dbt run` with views, `dbt run` with tables, `dbt test`, etc), iterating over each underlying component until basic functionality is up & running. The user docs will need to be their friend here.
4. Start using `pytest` to run the suite of basic functional tests. Those test cases will ensure that this plugin is feature-complete at a baseline level.
5. Your adapter is ready to share and start using!
6. Continue developing additional standard features (opt-in test cases within `dbt-tests-adapter`), plus your own database-specific functionality (custom test cases using the same framework)

cc @dataders I know you have thoughts here too!

Also: I think this section may want to live in the README at the root of this repo, rather than the cookie-cutter created README here. It's really describing the process of developing the new adapter, right after they've run the scaffold, before they have something they can share with colleagues / the world.

Reply:
This guidance might also want to live in our companion docs: https://docs.getdbt.com/docs/contributing/building-a-new-adapter

Reply:
@jtcohen6 would we want to add this to the building a new adapter page or would this be a better addition to https://docs.getdbt.com/docs/contributing/testing-a-new-adapter